wbDefinitionsFO76.pas

```diff
@@ -12672,8 +12800,30 @@ procedure DefineFO76g;
       wbFloat('Sky Scale'),
       wbFloat('Middle Gray')
     ], cpNormal),
+    wbStruct(FNAM, 'HDR', [
+      wbFloat('Eye Adapt Speed'),
+      wbFloat('Tonemap E'),
+      wbFloat('Bloom Threshold'),
+      wbFloat('Bloom Scale'),
+      wbFloat('Auto Exposure Max'),
+      wbFloat('Auto Exposure Min'),
+      wbFloat('Sunlight Scale'),
+      wbFloat('Sky Scale'),
+      wbFloat('Middle Gray'),
+      wbFloat('Unknown')
+    ], cpNormal),
     wbArray(FNAM, 'Unknown', wbFloat('Unknown')),
```

Comment:
Are there really two FNAMs right after each other? Or did you just forget to delete the old definition?

Reply:
Nope I forgot to remove it. Fixing thank you

Reply:
fixed in the commit below